scripts/install.sh

```diff
@@ -38,3 +38,6 @@ mkdir -p ../build/@digipolis-gent/modal;
 cp -R ./node_modules/@digipolis-gent/modal/dist/index.js ../build/@digipolis-gent/modal;
 cp -R ./node_modules/gent_styleguide/build/styleguide ../build;
 ./node_modules/.bin/gulp build;
+
+echo "Removing gent_base 'source' directory...";
+rm -rf ../source;
```

Comment:
Lokaal blijft het handig om de source niet te verwijderen tijdens het install script, ook bij bijv. een git checkout versie van het thema waarbij het install script gerund wordt. Is het misschien een idee om een install-prod.sh te maken waarbij install.sh ingeladen wordt + extra commands zoals in deze commit?

Reply:
Dat is een idee, maar dan moeten we op één of andere manier zeker zijn dat die install-prod.sh telkens bij `composer install` wordt uitgevoerd wanneer deze op onze deploy server wordt uitgevoerd, want momenteel wordt deze install.sh als post-install-script van composer uitgevoerd overal waar gent_base wordt gebruikt. Als we dat aanpassen loopt dat script ook nog steeds bij jullie lokaal...